File: tests/fixtures/python/JdkArrayCloneCopyRange.py
from __future__ import annotations


class JdkArrayCloneCopyRange:

    def __init__(self, values: list[str]) -> None:
        self.values = list(values)

    def remaining(self, offset: int) -> list[str]:
        return list(self.values[offset:len(self.values)])

File: tests/fixtures/java/JdkArrayCloneCopyRange.java
import java.util.Arrays;

public class JdkArrayCloneCopyRange {
    private final String[] values;

    public JdkArrayCloneCopyRange(String[] values) {
        this.values = values.clone();
    }

    public String[] remaining(int offset) {
        return Arrays.copyOfRange(values, offset, values.length);
    }
}
